### PR TITLE
Fixed blog issues, see description. Fixed grammar errors.

### PR DESCRIPTION
**Pull request by Sanyi0001**
Fixed grammar and spelling errors, unified every field description to start with a capital letter. Now the old values will be remembered and displayed correctly. Now the old select values will be remembered and displayed correctly. Email is verified using RFC and DNS verifications. Additionally, the selected values from the blog will be stored correctly in the corresponding variable.

### CODE REVIEW

**Review by awqfrxx — APPROVED** [review]
Very nice Sanyi.
Maybe a small thing, it might be nice to be consistent with brackets and spacing between them.
`value="{{$sdg->id}}" {{ (old('sdg') == $sdg->id ? "selected":"") }}>{{ $sdg->name }}</option>
`

**Review by avanheeren — APPROVED** [review]

File: resources/views/blogs/create.blade.php [hunk 1]
```diff
@@ -19,9 +19,10 @@
                         <div class="select is-fullwidth">
                             <select class="select-education @error('program') is-danger @enderror" type="text"
                                     id="program" name="program">
-                                <option value="{{ old('none') }}">None</option>
+                                <option selected value> None</option>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: what does value do here ?

File: resources/views/blogs/create.blade.php [hunk 2]
```diff
@@ -19,9 +19,10 @@
...
                                 @foreach ($programs as $program)
-                                    <option value={{ $program->name }}>{{ $program->name }}</option>
+                                    <option
+                                        value="{{$program->id}}" {{ (old('program') == $program->id ? "selected":"") }}>{{ $program->name }}</option>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: maybe put an enter after the option open tag, will be easier to read

File: resources/views/blogs/create.blade.php [hunk 3]
```diff
@@ -40,9 +41,11 @@
                         <div class="select is-fullwidth">
                             <select class="select-activity @error('activity') is-danger @enderror" type="text"
                                     id="activity" name="activity">
-                                <option value="{{ old('none') }}">None</option>
+                                <option selected value> None</option>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: value?

File: resources/views/blogs/create.blade.php [hunk 4]
```diff
@@ -103,11 +109,12 @@
...
                                     id="business_operation"
-                                    name="business_operation" value="{{ old('business_operation') }}">
-                                <option value="{{ old('none') }}">None</option>
+                                    name="business_operation">
+                                <option selected value> None</option>
+                                //TODO plural -> singular
                                 @foreach ($business_operation as $business_operation)
                                     <option
-                                        value={{ $business_operation -> name }}>{{ $business_operation -> name }}</option>
+                                        value="{{$business_operation->id}}" {{ (old('business_operation') == $business_operation->id ? "selected":"") }}>{{ $business_operation   ->name }}</option>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: after "{{" and before the "}}" there should be a space if I look to other of those blade things.

File: resources/views/blogs/create.blade.php [hunk 5]
```diff
@@ -176,8 +184,8 @@
             <div class="field-body">
                 <div class="field">
                     <div class="control">
-                        <textarea class="textarea @error('impact') is-danger @enderror" name="impact"
-                                  placeholder="Explain you impact"></textarea>
+                        <textarea class="textarea @error('impact') is-danger @enderror" name="impact" id="impact"
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php:191]: Do you use this id anywhere ?

**Review by Koreyoshi159 — CHANGES_REQUESTED** [review]
almost

File: resources/views/blogs/create.blade.php (hunk 5 above)
Comment (Sanyi0001) [reply resources/views/blogs/create.blade.php:191]: Yes

File: resources/views/blogs/create.blade.php (hunk 2 above)
Comment (Sanyi0001) [reply resources/views/blogs/create.blade.php]: Could you please clarify?

Comment (Koreyoshi159) [reply resources/views/blogs/create.blade.php]: Uhm not in text xD maybe give me a call xD

File: resources/views/blogs/create.blade.php (hunk 5 above)
Comment (Koreyoshi159) [reply resources/views/blogs/create.blade.php:191]: Where xD

**Pull request merged**